lib/creator/yeoman/utils/entry.js

```diff
@@ -0,0 +1,50 @@
+const Input = require('webpack-addons').Input;
+
+module.exports = (self, answer) => {
+	return new Promise( (resolve) => {
```

Comment:
http://bluebirdjs.com/docs/anti-patterns.html#the-explicit-construction-anti-pattern I'm sure you're refering to this, but I'm not sure how to make this async without using the antipattern. If you could elaborate that would be great. Using a promise here to avoid breaking inquirer. If you remove the promise the prompts get all invoked at once